### ISSUE TITLE
Application isn't running

### ISSUE THREAD

**Issue by FezVrasta** [timomeh/Dunkl.app#2]
Hi, almost once a week I get this error.
I have to manually start Dunkl again to make it work.

![image](https://cloud.githubusercontent.com/assets/5382443/25771821/b86dc0f6-325c-11e7-9d31-9ef82fd86751.png)

**Comment by timomeh** [comment]
As of Jan 5th, I don't maintain this app anymore. I wanted to add the final deprecation notice when the flux version gets out of beta, but I totally forgot about it.

**Comment by FezVrasta** [comment]
Flux version?

**Comment by timomeh** [comment]
Dark mode is back since f.lux 39.6-beta. Now f.lux is 39.94

**Comment by FezVrasta** [comment]
Oh I see, they fixed it on F.lux...

Umh, I think this app is still valuable because now there's the built-in night shift in macOS and I got rid of F.lux all together...

**Comment by timomeh** [comment]
Yep, I agree. But not the way I developed it. :D I just threw in a bit of AppleScript and manually added the plist so it runs in the background. Really a pain in the ass to maintain.

I want to get back to it as a side project, as a real macOS App.
